client/galaxy/scripts/mvc/dataset/dataset-edit-attributes.js

```diff
@@ -172,6 +172,14 @@ var View = Backbone.View.extend({
                     onclick: function() {
                         self._submit("datatype", form);
                     }
+                }),
+                submit_datatype_sniff: new Ui.ButtonIcon({
+                    tooltip: _l("Sniff the datatype and change it."),
+                    title: _l("Sniff datatype"),
```

Comment:
Can we go with something like 'Detect datatype' instead? Sniff could be confusing.

Reply:
Absolutely. Changed every 'sniff' to 'detect'. Thanks.